jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/quartz_tables_db2.sql

```diff
@@ -9,7 +9,7 @@ is_durable varchar(1) not null,
 is_nonconcurrent varchar(1) not null,
 is_update_data varchar(1) not null,
 requests_recovery varchar(1) not null,
-job_data blob(2000),
+job_data blob,
```

Comment:
That change is due to the following issue back in 2014: https://issues.redhat.com/browse/RHBPMS-1948?attachmentViewMode=list

Looking at the code, it was fixed (blob size 2000 was removed) but reverted back by mistake when upgrading to Quartz version 2.x. 

Therefore, this change is needed. 

Just to avoid this from happening again, I raised a PR on Quartz team to fix this issue so it won't happen again (though no answer yet so far).

PR on Quartz: https://github.com/quartz-scheduler/quartz/pull/559

Reply:
So it is still not fixed in Quartz 2.x? I would have thought that it was reverted back because the new version can cope with the limit :)

Reply:
I believe it was because we didn't raise any PR on Quartz repo by that time and we overrode our fix when we upgraded to new 2.x version afterwards - but this is only my guess :)

as you can see, they still have this blob(2000) limit in their script https://github.com/quartz-scheduler/quartz/blob/1b865546a043a1ffd9c3ddd345f94608ceccbd28/quartz-core/src/main/resources/org/quartz/impl/jdbcjobstore/tables_db2_v8.sql#L53

Reply:
Ah, I get it now, so we just copied it :) Then I'm fine with that.